tests/configure/test_config.py

```diff
@@ -57,42 +57,45 @@ def test_function(x):
 
 
 class TestDatabricksConfig(object):
-    databricks_config_from_password = config.DatabricksConfig.construct_from_password(
-        'test-host', 'test-user', 'test-password')
-    databricks_config_from_token = config.DatabricksConfig.construct_from_token(
-        'test-host', 'test-token')
+    databricks_config_from_password = config.DatabricksConfig()
+    databricks_config_from_password.update_with_password(config.DEFAULT_SECTION, 'test-host',
+                                                         'test-user', 'test-password')
+    databricks_config_from_token = config.DatabricksConfig()
+    databricks_config_from_token.update_with_token(config.DEFAULT_SECTION, 'test-host',
+                                                   'test-token')
 
     def test_is_valid_true(self):
         assert self.databricks_config_from_password.is_valid
         assert self.databricks_config_from_token.is_valid
 
     def test_is_valid_false(self):
         databricks_config = config.DatabricksConfig()
-        assert not databricks_config.is_valid
+        assert not databricks_config.is_valid(config.DEFAULT_SECTION)
```

Comment:
Do we need `DEFAULT_SECTION` everywhere? Can we simply call `assert not databricks_config.is_valid()`

Reply:
IMO, the explicit parameter adds some clarity here.